libs/workspace-manager/src/lib/data-access/workspace-manager-api.service.ts

```diff
@@ -0,0 +1,24 @@
+import { HttpClient } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import { Directory } from '@angular-cli-gui/shared/data';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class WorkspaceManagerApiService {
+  private readonly http = inject(HttpClient);
+  private readonly API_URL = '/api/workspace-manager';
+
+  getHomeDir(): Observable<string> {
+    return this.http.get(`${this.API_URL}/homedir`, {
+      responseType: 'text',
```

Comment:
it should also work without specifying this no?

Reply:
since the return type is string i get an error because the 'regular' .get overload expects a JSON format.
we can either solve it like that in the FE or adjust the api to return `{path: string}`

Reply:
<img width="663" alt="image" src="https://user-images.githubusercontent.com/60091317/223358984-ae81b08e-f0a4-467f-9316-59ee84f712b6.png">